config/config.yml

```diff
@@ -0,0 +1,45 @@
+# App configuration
+VERSION: "0.0.3"
+EXCLUDED_FILES:
+  [
+    ".*",
+    "Thumbs.db",
+    ".DS_Store",
+    "._.DS_Store",
+    ".localized",
+    "desktop.ini",
+    "*.pyc",
+    "*.swx",
+    "*.swp",
+    "*~",
+    "~$*",
+    "NULLEXT",
+  ]
+APP_NAME: "frdr-encryption-client"
+APP_AUTHOR: "Compute Canada"
+TIMEZONE: "America/Regina"
+GLOBUS_DATA_PUBLICATION_CLIENT_ID: "7dbc22b3-60c1-4e75-b148-3ee89f95792d"
+GLOBUS_DATA_PUBLICATION_CLIENT_NAME: "DataPublication Client v0.1"
```

Comment:
OK here! I finally got to this file, shame I didn't see it sooner. Here is the place to specify those configs I mentioned earlier.

Reply:
Those configs have been added to this file.